sig-cloud-provider/2020-annual-report.md

```diff
@@ -0,0 +1,85 @@
+
+We try hard to make sure we have presentation from each of the interested cloud providers.
+
+---
+## Current initiatives and project health
+_Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members._
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+[Feature: implement the BackendManager list](https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/144) fixes a long standing issue for people running clusters with nodes which are either not routable or at least not close to one another.
+[Fix flag passing in CCM](https://github.com/kubernetes/kubernetes/pull/98210) fixes a significant problem/cost most cloud providers experienced trying to remain current on their CCM dependency.
+[Extending Apiserver Network Proxy to handle traffic originated from Node network](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2025-extend-konnectivity-for-both-directions) is an interesting and difficult initiative around build out K8s where the control plane and data plane are remote.
+
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+[Promoting Cloud Provider Labels to GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/837-cloud-provider-labels) to GA
+
+[API Server Network Proxy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1281-network-proxy) to Beta
+
+[Reporting Conformance Test Results to Testgrid](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2390-reporting-conformance-test-results-to-testgrid) ongoing.
+[Building Kubernetes Without In-Tree Cloud Providers](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/1179-building-without-in-tree-providers) to Alpha
+[Extending Apiserver Network Proxy to handle traffic originated from Node network](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2025-extend-konnectivity-for-both-directions)
+[Kubelet Credential Providers](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2133-kubelet-credential-providers) to Alpha
+[Leader Migration for Controller Managers](https://github.com/kubernetes/enhancements/issues/2436) to Alpha
+
+**What areas and/or subprojects does the group need the most help with?**
+
+We run the cloud provider extraction working group initiative. We need help engaging more with cloud providers. We also really need help dealing with the both the K/K "cluster" directory and resolving how we properly test K/K in the absense of a cloud provider.
+
+**What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?**
+
+Pretty long :(
```

Comment:
what do you attribute this to?

Reply:
Still trying to build up a team of multiple contributors for every cloud provider to help triage and handle these. We are slowly growing this and its getting better but more so with some cloud providers than others.